cli/plasmo/src/commands/init.ts

```diff
@@ -126,7 +129,7 @@ async function init() {
     "Your extension is ready in: ",
     chalk.yellowBright(projectDirectory),
     `\n\n    To start hacking, run:\n\n`,
-    `      cd ${packageName}\n`,
+    rawName === "." ? "" : `      cd ${packageName}\n`,
```

Comment:
This will lead to unexpected output on inputs like:

```
./
./.
././.
.. etc 

and stuff like
../thisDirectory

```

Maybe we should check whether the `projectDirectory` matches the `cwd`, instead, and have the `resolve` be our oracle`

Reply:
Got it, this is some edge case imo, but I will resolve it in a patch